File: octave/genpoiss.py
################################
# genpoiss.py
# Generira nakljucna stevila porazdeljena po  
# poissonovi porazdelitvi in jih shrani v datoteko.

import numpy

lambd = 1;
N=1000000;

D = numpy.random.poisson(lambd,N)
numpy.savetxt('numpyPoiss.txt', D, fmt='%d')

File: octave/genpoiss.m
%%%%%%%%%%%%%%%%%%%%
% genpoiss
% Generira nakljucna stevila porazdeljena po  
% poissonovi porazdelitvi in jih shrani v datoteko.
% Koda: Standardna Octave
% 

function genpoiss

  lambda = 1;
  N=1000000;

  D = poissrnd(lambda,N,1);
  save('-ascii', 'podatki/octavePoiss.txt', 'D')

end